### DISCUSSION TITLE
Halloween - Headless Horseman boss

### DISCUSSION THREAD

**Discussion by shaszun** [chromiecraft/chromiecraft#6224] [Suggestions]
Last year this event boss wasn't available. My guess is, because of missing level 60 loot table. But this year with level 70 progress I don't see reasons to not get this boss with the appopriate loot.

**Comment by Hihilist** [comment]
Sorry to rain on your parade but if nothing has changed then:
![obraz](https://github.com/chromiecraft/chromiecraft/assets/71841180/5273b00a-99bb-4a7d-a6eb-9c8fd1742549)

**Reply by shaszun** [reply]
I want to hear/read the stupid reasoning about that statement. :)

**Comment by shaszun** [comment]
Why the outland boss for the summer festival had loot?
